pkg/services/accesscontrol/accesscontrol.go

```diff
@@ -15,6 +15,9 @@ type AccessControl interface {
 
 	// Middleware checks if service disabled or not to switch to fallback authorization.
 	IsDisabled() bool
+
+	// RegisterFixedRole adds a new fixed role to built-in roles
+	RegisterFixedRole(ctx context.Context, role RoleDTO, builtInRoles ...string) error
```

Comment:
```suggestion
	RegisterFixedRole(ctx context.Context, role RoleDTO, builtInRoles ...models.RoleType) error
```

Reply:
@sakjur said something similar in the doc. However, this does not include "Grafana Admin".
```go
type RoleType string

const (
	ROLE_VIEWER RoleType = "Viewer"
	ROLE_EDITOR RoleType = "Editor"
	ROLE_ADMIN  RoleType = "Admin"
)
```
Options I can think of:
* Create a validate function for the string
* Create a type builtInRoleType that adds "Grafana Admin" to models.Type
* Create another function RegisterGrafanaAdminFixedRole